packages/server/lib/controllers/config.controller.ts

```diff
@@ -131,7 +132,8 @@ class ConfigController {
                         provider: config.provider,
                         scripts: activeFlows.length,
                         connection_count: Number(config.connection_count),
-                        creationDate: config.created_at
+                        creationDate: config.created_at,
+                        missing_fields_count: config.missing_fields.length
```

Comment:
why not returning the missing fields and doing the count in the FE?

Reply:
I mostly just figured since we're only indicating if there is or isn't an error that it'd be better and more compact to just return a count there. I'm down if you think we should swap it for the full array.